### ISSUE TITLE
Emissions restriction

### ISSUE THREAD

**Issue by Almorca** [osmandapp/OsmAnd#16218]
The number of low-emission zones where vehicles are restricted according to their pollution levels is increasing daily.

In OSM there are examples like https://www.openstreetmap.org/relation/7966961

It's common to use a motor_vehicle:conditional tag with vale no @ (emissions < euro_5 )

It would be useful if, just as osmand already monitors weight and length restrictions, it could also take into account the European Emission standars ( https://en.wikipedia.org/wiki/European_emission_standards ).

**Comment by vshcherb** [comment]
We need to display if route enters emission zone restriction so people could choose to avoid

**Comment by Kovoschiz** [comment]
That's not accurate. It needs to be eg `toll:conditional=yes @ (*)` as they are allowed by paying, or purchasing a pass. https://www.slimnaarantwerpen.be/en/LEZ/check-your-vehicle/conditions-for-admission-as-of-1/1/2020 
As the tagging in general isn't clear (eg only certain pollutants are restricted), warning for the `boundary=` alone will be enough at first.

**Comment by Zweitaktmotor** [comment]
Yes, boundary would be good, but also an option to avoid/calculate alternative routes.

Sorry, I don't know much about the programming behind it and apprreciate all the volunteer efforts.
